src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Server/ServerHandler.cs

```diff
@@ -91,5 +92,23 @@ private int GetServerMinMemory()
         {
             return configService.GetServerSmoConfig(server, configService.MinServerMemoryPropertyNumber).ConfigValue;
         }
+
+        private void UpdateServerProperties(InitializeViewRequestParams viewParams, ServerInfo serverObj)
+        {
+            if (viewParams != null)
+            {
+                ConnectionInfo connInfo = this.GetConnectionInfo(viewParams.ConnectionUri);
+                ServerConnection serverConnection = ConnectionService.OpenServerConnection(connInfo, ObjectManagementService.ApplicationName);
+                this.server = new Server(serverConnection);
+                if (GetServerMaxMemory() != serverObj.MaxServerMemory)
+                {
+                    configService.UpdateConfig(serverConnection, configService.MaxServerMemoryPropertyNumber, serverObj.MaxServerMemory);
+                }
+                else if (GetServerMinMemory() != serverObj.MinServerMemory)
```

Comment:
Are we able to update both min memory and max memory in the same operation?  With this else statement we always skip min memory updates if there's a max memory update.